puts "Why does the following code..."


# def execute(block)
#   block.call
# end

#execute { puts "Hello from inside the execute method!" }

puts "Give us the following error when we run it?"

#block.rb1:in `execute': wrong number of arguments (0 for 1) (ArgumentError) from test.rb:5:in `<main>'

puts "If a method is to take a block, then we need to see the parameter preceed with an ampersand."